tests/wpt/metadata/css/cssom/MediaList2.xhtml.ini

```diff
@@ -1,4 +1,12 @@
-[MediaList2.xhtml]
-  [MediaList]
+[MediaList2.xhtml]	
+  [MediaList.mediaText]	
+    expected: FAIL	
+
+  [MediaList.length]	
```

Comment:
Do we know why these are regressing?

Reply:
I am not sure of the reason but these were failing from the start. Josh asked me to leave that for now and report it in a new issue. https://mozilla.logbot.info/servo/20190408#c16185606

Reply:
They're not regressing; the test file uses ownerNode before running these tests, so they never had a chance to run before now.